src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -1230,10 +1214,38 @@ export class StandaloneSearchInterface extends SearchInterface {
     // By using a setTimeout, we allow other possible code related to the search box / magic box time to complete.
     // eg: onblur of the magic box.
     setTimeout(() => {
-      this._window.location.href = `${link.protocol}//${link.host}${pathname}${link.search}${hash}${HashUtils.encodeValues(stateValues)}`;
+      this._window.location.href = `${link.protocol}//${link.host}${pathname}${link.search}${hash}${this.encodedHashValues}`;
     }, 0);
   }
 
+  private get encodedHashValues() {
+    const values = this.modelAttributesToIncludeInUrl;
+    const uaCausedBy = this.uaCausedBy;
```

Comment:
Could the fact that we're going to be still sending stuff in the URL for UA "bother" the client ? What was their original request for asking us to remove values from the URL when `usingLocalStorageHistory` ? 

Was it because "anything in the URL is causing problem" ?

Reply:
The comment was directed at the query in the url becoming out of sync as additional queries are performed. Here is the excerpt from Felix in the [jira](https://coveord.atlassian.net/browse/JSUI-2863):

> the url from the redirection does not get update whenever performing new queries. This means that the user can end up in confusing situation where the url and the query performed do not match up.